source/connect/favorite-connections/import-export-ui/import.txt

```diff
@@ -0,0 +1,57 @@
+.. _compass-import-connections-ui:
+
+==============================================
+Import Favorite Connections in |compass-short|
+==============================================
+.. default-domain:: mongodb
```

Comment:
Not sure if we need a blank line between the title and `.. default-domain::`

Reply:
Ah good catch! Updated.

Reply:
Not sure why, but on GH this isn't reflecting the new line that I added. 😳